numpy/random/_generator.pyx

```diff
@@ -4114,7 +4172,117 @@ cdef class Generator:
 
         return diric
 
-    # Shuffling and permutations:
+    def permuted(self, object x, *, axis=None, out=None):
+        """
+        Randomly permute (i.e. shuffle) `x` along axis `axis`.
+
+        Unlike `shuffle`, each slice along the given axis is shuffled
+        independently of the others.
+
+        If `axis` is None, the flattened array is shuffled.
```

Comment:
Should this comment and the next be moved to the Parameters section where they are better connected to their parameter?

Reply:
Done.